2200009/SWEA_14654_신용카드만들기2.py

```diff
@@ -0,0 +1,12 @@
+i_t = int(input())
+lst_start = ['3', '4', '5', '6', '9']
+for i_1 in range(i_t):
+    s_card = input()
+    s_newcard = ''
+    for s_1 in s_card:
+        if s_1 != '-':
+            s_newcard += s_1
+    if (s_newcard[0] in lst_start) * (len(s_newcard) == 16): 
```

Comment:
좋은 풀이네요! 👍 
```suggestion
    if (s_newcard[0] in lst_start) and (len(s_newcard) == 16): 
```
추가로 `*` (곱하기) 연산자는 참/거짓 boolean 자료형에서도 연산이 가능하여 지금 써주신 방법대로 작성해주셔도 괜찮습니다.
하지만, 코드의 가독성과 "Python스러운 코드"를 위해 `and`로 표현해주시면 더욱 좋을 것 같습니다 😄